setup.py

```diff
@@ -40,11 +40,19 @@
                        "{}/lib/".format(ucc_home)]
 plugin_libraries    = ["ucp", "uct", "ucm", "ucs", "ucc"]
 
+if '--oss' in sys.argv:
+  sys.argv.remove('--oss')
+  plugin_sources += ["src/torch_ucc_init_oss.cpp"]
+  name = "torch_ucc_oss"
+else:
+  plugin_sources += ["src/torch_ucc_init.cpp"]
+  name = "torch_ucc"
```

Comment:
this name is what should be used regardless of whether oss or not

Reply:
fixed